qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -1186,6 +1194,7 @@ def test_backlog(self):
 
                 for t in threads:
                     t.start()
+                    time.sleep(1)
```

Comment:
This sleep will prevent concurrent inference requests. The goal of this test case is to:
```
# Send 5 equal-length sequences in
# parallel and make sure they get completely batched into
# batch-size 4 inferences plus the 5th should go in the
# backlog and then get handled once there is a free slot.
```

Reply:
Ah... didn't think it clearly, updated, thanks!